odl/discr/discr_mappings.py

```diff
@@ -235,72 +230,52 @@ def _call(self, inp):
 
         Finally create the operator:
 
-        >>> coll_op = RawGridCollocation(funcset, grid, rn)
-        >>> func_elem = funcset.element(lambda x, y: x - y)
+        >>> coll_op = GridCollocation(funcset, grid, rn)
+
+        We define an example function with meshgrid vectorization:
+
+        >>> def func(mg_tuple):
```

Comment:
Yes, I realized that with the way the arrays or tuples work, this code will work as expected in fact. I'll change that.